static/src/js/components/Header/Header.jsx

```diff
@@ -157,6 +158,7 @@ const Header = () => {
                 >
                   User Guide
                 </Button>
+
```

Comment:
Not a huge fan of these empty spaces between the components. I think we are a little inconsistent with it but my vote would be no extra spaces 😅 I think thats what we do in most places.